### ISSUE TITLE
Zipped shapefile should not be supported

### ISSUE THREAD

**Issue by jingsam** [mapbox/mapbox-file-sniff#41]
In https://github.com/mapbox/mapbox-file-sniff/blob/master/index.js#L51
It just make sure that the file is a zip file, can not guarantee it is a zipped shapefile

What's worse, tilelive-omnivore use this library to construct a protocol:
```
omnivore://./path/to/a.zip
```
Then this a.zip was write to a mapnik XML. But the truth is mapnik does not support zipped shapefile!

**Comment by mapsam** [comment]
I agree - if we are just checking for a zip file that can let a lot of cruft through. Since we are decompressing some files (serialtiles & tif+gz) can we unzip shapefiles as well and take a look at their contents?
